test/beetle/subscriber_test.rb

```diff
@@ -225,7 +225,7 @@ def setup
   end
 
 
-  class DeadLetteringCallBackExecutionTest < Minitest::Test
+  class QueuePropertiesCallBackExecutionTest < Minitest::Test
```

Comment:
The test class has little to do with the `QueueProperties`, so I think we could just keep the old name.

Reply:
changed